src/System/Console/GetOpt/Generics.hs

```diff
@@ -270,6 +310,8 @@ combine (ParseErrors e) _ = ParseErrors e
 combine (Unset _) x = x
 combine (FieldSuccess _) (ParseErrors e) = ParseErrors e
 combine (FieldSuccess a) (FieldSuccess b) = FieldSuccess (_accumulate a b)
+combine PositionalArguments _ = PositionalArguments
```

Comment:
Well, `PositionalArguments` should never occur as the second argument. I still think it's best to come up with something that makes the most sense. And yes, is my mind `PositionalArguments` should always trump. I'll change this.